src/llmq/utils.h

```diff
@@ -19,9 +19,11 @@ class CBlockIndex;
 class CDeterministicMN;
 class CDeterministicMNList;
 class CDeterministicMNManager;
-using CDeterministicMNCPtr = std::shared_ptr<const CDeterministicMN>;
+class CMasternodeMetaMan;
 class CSporkManager;
 
+using CDeterministicMNCPtr = std::shared_ptr<const CDeterministicMN>;
```

Comment:
nit: that's still forward declaration, not sure that empty line here is reasonable but nevermind
```suggestion
using CDeterministicMNCPtr = std::shared_ptr<const CDeterministicMN>;
```